src/overlay/Peer.cpp

```diff
@@ -1868,7 +1991,103 @@ Peer::PeerMetrics::PeerMetrics(VirtualClock::time_point connectedTime)
     , mDuplicateFloodMessageRecv(0)
     , mUniqueFetchMessageRecv(0)
     , mDuplicateFetchMessageRecv(0)
+    , mTxHashReceived(0)
     , mConnectedTime(connectedTime)
 {
 }
+
+bool
+Peer::supportsAdverts() const
+{
+    return ((mApp.getConfig().OVERLAY_PROTOCOL_VERSION >=
```

Comment:
I added the config `ENABLE_PULL_MODE`. What would be a good way to tell the peer that the node wants to enable/disable pull mode? i first looking into adding a filed in `Hello` in `Stellar-overlay.x`, but I didn't do it since it seemed a bit too permanent & I wasn't sure if this is the right place for it.

Reply:
oh I see. Hmm, maybe we can use the value in the Auth message, I see it's unused. It's kind of ugly, but at the same time this config is transient for the time of the rollout, so introducing a new temporary message version seems annoying.

Reply:
Done.